tests/CMakeLists.txt

```diff
@@ -1,29 +1,39 @@
 set(_tests_in_build_tree ${PROJECT_BINARY_DIR}/${PYMOD_INSTALL_FULLDIR}/..)
 
-configure_file(
-  ${CMAKE_CURRENT_SOURCE_DIR}/test_vampyr3d.py
-  ${_tests_in_build_tree}/test_vampyr3d.py
-  COPYONLY
+list(APPEND _tests
```

Comment:
I think it's fine to glob here and then create _links_ to the files:
```cmake
# we glob the Python test files and let CMake add a rule such that
# the glob is repeated every time we rebuild.
# This is NOT RECOMMENDED by CMake
# (https://cmake.org/cmake/help/v3.16/command/file.html#filesystem) but you only
# live once!
file(
  GLOB
    _vlx_pytests
  LIST_DIRECTORIES
    FALSE
  CONFIGURE_DEPENDS
  RELATIVE
    ${CMAKE_CURRENT_LIST_DIR}
  test_*.py
  )
# append conftest.py to the list
list(APPEND _vlx_pytests conftest.py)

foreach(_f IN LISTS _vlx_pytests)
  # link the Python files under the build folder
  file(
    CREATE_LINK
      ${CMAKE_CURRENT_LIST_DIR}/${_f}
      ${PROJECT_BINARY_DIR}/${PYMOD_INSTALL_FULLDIR}/tests/${_f}
    COPY_ON_ERROR
    SYMBOLIC
    )

  if(${_f} MATCHES "conftest.py")
    break()
  endif()

  # extract the name and label for the test from the file name
  # by removing the "test_" prefix and the ".py" suffix
  string(REGEX REPLACE "test_(.*).py" "\\1" _name ${_f})
  add_test(
    NAME
      ${_name}
    COMMAND
      ${MPIEXEC_EXECUTABLE} ${MPIEXEC_NUMPROC_FLAG} 1 ${Python_EXECUTABLE} -m pytest -v -s --pyargs veloxchem -k test_${_name}
    WORKING_DIRECTORY
      ${PROJECT_BINARY_DIR}/${PYMOD_INSTALL_FULLDIR}/..
    )

  set_tests_properties(${_name}
    PROPERTIES
      LABELS
        "integration;${_name}"
    )
endforeach()


# install the current folder
install(
  DIRECTORY
    ${CMAKE_CURRENT_LIST_DIR}/
  DESTINATION
    ${PYMOD_INSTALL_FULLDIR}/tests
  PATTERN "CMakeLists.txt" EXCLUDE
  )
```

This will pick up changes to the tests.

Reply:
Ok, I think I managed to import this correctly